### DISCUSSION TITLE
Login as user

### DISCUSSION THREAD

**Discussion by palicepta** [dani-garcia/vaultwarden#3809] [Q&A] [answered]
Dear All,

How I can login as another user by using an admin account?

Thank you in advance.

**Comment by BlackDex** [comment]
You can't, and that will never be an option.
Also, i would never use a password manager when this would be possible.

**Comment by palicepta** [comment]
For example
<img width="1014" alt="image" src="https://github.com/dani-garcia/vaultwarden/assets/112872766/ec0645b5-2c67-4099-afd1-a6188c51ceea">

I see the policy Master password reset. But how to reset password I don't find.

Anybody know something about this?

**Reply by BlackDex** [reply]
Read https://bitwarden.com/help/account-recovery/

**Comment by palicepta** [comment]
Dear [BlackDex](https://github.com/BlackDex),

> Also, i would never use a password manager when this would be possible.

We are using the password manager for our team and if someone will leave the team, we should have the access to his account.

**Reply by BlackDex** [reply, marked as answer]
I would dispute that. You should just make sure they put everything company/team wide into the organization. And his personal passwords should stay personal. You should just delete his account and be done with it.

You can also prevent personal passwords if I'm correct, so that could be a solution.
